solutions/swb-app/src/projects/disassociateUserFromProjectRequest.ts

```diff
@@ -3,12 +3,12 @@
  *  SPDX-License-Identifier: Apache-2.0
  */
 
-import { z } from 'zod';
+import { z } from '@aws/workbench-core-base';
 
 // eslint-disable-next-line @rushstack/typedef-var
 export const DisassociateUserFromProjectRequestParser = z
   .object({
-    userId: z.string(),
+    userId: z.string().userId().required(),
     projectId: z.string()
```

Comment:
nit: .projId().required(),

Reply:
for the record this is more scope creep but you are right to call it out that it needs to be fixed!